packages/rum-core/src/performance-monitoring/transaction-service.js

```diff
@@ -164,13 +155,6 @@ class TransactionService {
       if (perfOptions.pageLoadSampled) {
         tr.sampled = perfOptions.pageLoadSampled
       }
-      /**
-       * Retriving the name before transaction ends should reflect
```

Comment:
The same here, we need to set the correct name at the start of the transaction.

Reply:
Its already set as part of apm-base https://github.com/elastic/apm-agent-rum-js/blob/b073f7f7a4c2ffdc248b8c5d6eff5a13c5eb0afd/packages/rum/src/apm-base.js#L75.  I really have no clue why we need this like 3 places if apm-base is the once that sets the listener and registers for capturing the page-load transaction.

Reply:
I think we should remove the one in the apm-base, to avoid having the logic split in multiple places and keep the page load logic in transaction service!

Reply:
Sure, in that case lets' have it here.

Reply:
Would you please add a comment to make it clear for future!

Reply:
Sure thing.